code/modules/projectiles/ammo_datums.dm

```diff
@@ -690,13 +697,20 @@
 /datum/ammo/bullet/revolver/mateba/highimpact
 	name = ".454 heavy high-impact revolver bullet"
 	debilitate = list(0,2,0,0,0,1,0,0)
-	penetration = ARMOR_PENETRATION_TIER_2
+	penetration = ARMOR_PENETRATION_TIER_1
 	flags_ammo_behavior = AMMO_BALLISTIC
 
+/datum/ammo/bullet/revolver/mateba/highimpact/ap
+	name = ".454 high-impact armor piercing revolver bullet"
```

Comment:
```suggestion
	name = ".454 heavy high-impact armor piercing revolver bullet"
```

it stuns just like the parent, should also be heavy for consistency not only with mateba but also m44 heavy IMO